src/api/spec/README.md

```diff
@@ -168,32 +155,34 @@ Make sure you enable VCR in the test metadata like this:
   end
 ```
 
-#### Remove all cassettes and run the test again before you commit
+#### Remove All Cassettes and Run the Test Again Before You Commit
+
 Before you finally commit your test, you should remove the generated cassettes and run your test again.
 This ensures that only by the test needed responses are included in the cassette and nothing more.
 You can also review the cassette manually (but **NEVER** edit them manually)!
 
-### Shared examples
-To DRY our tests we use in rare situations [shared examples](https://www.relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples).
+### Shared Examples
+
+To DRY our specs we use in rare situations [shared examples](https://www.relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples).
 You should only use shared examples where you have the exact same functionality (e.g. package/project or user/group tab).
-Otherwise these tests get fast hard to refactor and review.
+Otherwise, these specs are a nightmare to refactor and review.
 In our experience, shared examples are used mainly for controllers. Since models are pretty different from each other, they (usually) do not share much logic.
 
-### Continuous Integration
```

Comment:
This is easy to know by looking at any PR... I don't see the point in having this documented here.